net-mail/queue-repair/files/python3.patch

```diff
@@ -0,0 +1,239 @@
+--- a/queue_repair.py 2003-10-22 23:54:13.000000000 +0800
++++ b/queue_repair.py 2020-01-17 18:21:18.834455813 +0800
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python
++#!/usr/bin/python3
```

Comment:
You can call `python_fix_shebang` to automatically fix this one. 
https://dev.gentoo.org/~mgorny/python-guide/helper.html